### ISSUE TITLE
Allow cards to be pinned in the homefeed

### ISSUE THREAD

**Issue by wschella** [ZeusWPI/hydra-android#259]
This is more a discussion kinda thing.
Would it make sense to allow users to pin certain cards in the homefeed?
Examples might include:
 - Current resto menu
 - Current class schedule

**Comment by feliciaan** [comment]
We can increase the priority of certain cards 👍

**Comment by niknetniko** [comment]
I think a combination of other features enable something like this already:
- The priorities should be set in such a way that relevant cards are at the top. For example, during lunch hours, the resto cards get a slightly higher priority and the calendar gets a lower priority (and vice versa during the rest of the day). Perhaps the priorities need a re-calibration?
- The user can swipe away cards they are done with. For example, if a WiNA event is at the top, but the user doesn't attend it, they can swipe it away. This works for all cards.
- If the user wants to only view the resto card for example, they can directly launch the resto portion (using app shortcuts on Android 7.1 or higher)

TL;DR: I'm not sure if this adds something that's not already possible

**Comment by niknetniko** [comment]
As mentioned above, the goal is for the "automatic" sorting to be smart enough to put relevant cards at the top (e.g. during lunch ours, the resto cards get a boost in importance), which together with the ability to customise which cards are shown should provide this functionality.

If, for example, resto cards are consistently not at the top during lunch ours, this is a bug; feel free top file an issue in that case.
